RR_manuscript/RR.Rmd

```diff
@@ -1972,7 +1972,7 @@ ms_meta_high <- round(ms_sd_max * meta_high * 1.96, 1)
 
 
 For example, for those analyses that investigated the duration of vowels (e.g. the duration of the stressed vowel in *Banáne*), the reported duration measures exhibit standard deviations that range from `r ms_sd_min` to `r ms_sd_max` ms.
-These standard deviations allow us to convert the meta-analytic estimates into milliseconds by multiplying those values with the meta-analytic estimates standard units.
+These standard deviations allow us to convert the meta-analytic estimates into milliseconds by multiplying those values with the meta-analytic estimated standard units.
```

Comment:
TO me brain "meta-analytic estimates standard units" and "meta-analytic estimated standard units" are not the same thing. I will change it to "the standard unit values of the meta-analytic estimates"